frontend/src/components/project/ProjectContent.js

```diff
@@ -357,3 +401,43 @@ function CollaborateContent({ project, texts }) {
     </>
   );
 }
+
+function DiscussionPreview({ latestParentComment, discussionTabLabel, locale, project }) {
+  const classes = useStyles();
+  return (
+    <>
+      <Link
+        href={`${getLocalePrefix(locale)}/projects/${project.url_slug}#comments`}
+        underline="none"
+      >
+        <div className={classes.discussionPreview}>
+          <div className={classes.topSectionDiscussionPreview}>
+            <Typography
+              display="inline"
+              color="primary"
+              className={classes.headingDiscussionPreview}
+            >
+              {discussionTabLabel}
+            </Typography>
+            <UnfoldMoreIcon color="secondary" className={classes.iconDiscussionPreview} />
+          </div>
+          <div className={classes.commentDiscussionPreview}>
+            <Avatar
+              className={classes.commentAvatarDiscussionPreview}
+              src={
+                latestParentComment.author_user.image
+                  ? getImageUrl(latestParentComment.author_user.image)
+                  : getImageUrl(latestParentComment.author_user.thumbnail_image)
+              }
+            />
+            <Typography className={classes.commentTextDiscussionPreview}>
+              <Truncate lines={3} ellipsis={"..."}>
+                {latestParentComment.content}
+              </Truncate>
+            </Typography>
+          </div>
```

Comment:
For this it would make sense to use `frontend/src/components/communication/Posts`. This includes all the functionality necessary for displaying comments properly, e.g. parsing links and user-tagging. You could then adapt this function to also fit your needs (while keeping it working as is for the current use-cases), e.g. by adding a variable `truncate` which could indicate whether and after how many lines a text should be truncated.

Reply:
The Posts component is now used for the DiscussionPreview! Using the component indeed got rid of the bug mentioned above :slightly_smiling_face: